pkg/volume/scaleio/sio_util.go

```diff
@@ -107,6 +109,12 @@ func validateConfigs(config map[string]string) error {
 	if config[confKey.system] == "" {
 		return systemNotProvidedErr
 	}
+	if config[confKey.storagePool] == "" {
```

Comment:
@jsafrane another clarification on that point.  The code was substituting `"default"` for those values when they were not provided.  However, production deployments are most likely not using that value anyway.  Backward compat is ok.  Will still add it to release note.